### PR TITLE
[search] Update name score with IATA airport code.

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

**Comment by mpimenov** [comment]
A test?

**Comment by tatiana-yan** [comment]
>A test?

done

File: search/search_integration_tests/processor_test.cpp [hunk 1]
```diff
@@ -1503,5 +1526,32 @@ UNIT_CLASS_TEST(ProcessorTest, CuisineMetadataTest)
...
+    TRules rules{ExactMatch(countryId, vko)};
+    TEST(ResultsMatch("vko ", "en", rules), ());
+  }
+
+  {
+    TRules rules{ExactMatch(countryId, svo)};
+    TEST(ResultsMatch("svo ", "en", rules), ());
+  }
+}
+
```
Comment (mpimenov) [inline search/search_integration_tests/processor_test.cpp]: пустые лишние

Comment (tatiana-yan) [reply search/search_integration_tests/processor_test.cpp]: fixed

**Pull request merged**